test/TestUtils.kt

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
Let's rename this file to `Utils.kt`.

A particular naming practice that we always adhere to (which is not always the standard!) is to take advantage of our scoped names - ie: instead of `test/TestUtils.kt`, we would say, just `test/Utils.kt`.

We sacrifice having a "globally unique" identifier in favour of conciseness, canonical-ness and not repeating words.

Reply:
Understood. This has been implemented.